docs/tutorials/tutorial_3k_tcr.md

```diff
@@ -604,35 +604,54 @@ ir.pl.clonotype_imbalance(
 ## Integrating gene expression
 ### Clonotype imbalance among cell clusters
 
-Leveraging the opportunity offered by close integeration with scanpy, transcriptomics-based data can be utilized directly. As an example, using cell type annotation inferred from gene expression clusters, clonotypes belonging to CD4+ and CD8+ regional memory T-cells can be compared.
+Leveraging the opportunity offered by close integeration with scanpy, transcriptomics-based data can be utilized directly. As an example, using cell type annotation inferred from gene expression clusters, clonotypes belonging to CD8+ effector T-cells and CD8+ tissue-resident memory T cells, for example, can be compared.
 
 ```python
 freq, stat = ir.tl.clonotype_imbalance(
     adata,
     replicate_col="sample",
-    groupby="source",
-    case_label="Tumor",
-    control_label="Blood",
-    additional_hue="cluster",
+    groupby="cluster",
+    case_label="CD8_Teff",
+    control_label="CD8_Trm",
     inplace = False,
 )
-top_differential_clonotypes = stat.groupby("clonotype").agg({"logpValue": "max"}).reset_index().sort_values(by="logpValue", ascending=False).head(n=15).clonotype.tolist()
+top_differential_clonotypes = stat.groupby("clonotype").agg({"logpValue": "max"}).reset_index().sort_values(by="logpValue", ascending=False).head(n=5).clonotype.tolist()
```

Comment:
as the table is already ordered, this can be simpliefied to 
```
stat["clonotype"][:5].values
```

Reply:
True, this was only needed for the previous case, where we had two grouping factors.